public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -20,9 +19,28 @@ type Props = {
 };
 
 export function PluginDetailsBody({ plugin, queryParams, pageId }: Props): JSX.Element {
+  type Cell<T extends keyof Permission = keyof Permission> = CellProps<Permission, Permission[T]>;
```

Comment:
This type can be defined outside the component.

Reply:
Changed as requested.